lib/matplotlib/backends/backend_wx.py

```diff
@@ -1449,7 +1435,7 @@ def updateAxes(self, maxAxis):
             for menuId in self._axisId[maxAxis:]:
                 self._menu.Delete(menuId)
             self._axisId = self._axisId[:maxAxis]
-        self._toolbar.set_active(list(xrange(maxAxis)))
+        self._toolbar.set_active(range(maxAxis))
```

Comment:
This works without being a list? (I didn't look into where the call goes.)

Reply:
actually the whole class can be deprecated (see updated message at top / PR message).